game_of_life/game_o_life.py

```diff
@@ -31,6 +31,9 @@
 import sys
 
 import numpy as np
+
+from matplotlib import use as mpluse
+mpluse('TkAgg')
```

Comment:
Note that this should be called before importing pyplot. Otherwise, it has no effect.